src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -1,37 +1,44 @@
 #!/bin/bash
-
 set -euo pipefail
-config=${CONFIG:-}
+
+config="$(cat "${CONFIG:-}")"
```

Comment:
Couldn't this be an empty file-name causing `cat` to fail with an error?

Reply:
Good point! will fix that